src/Grpc/Directory.Build.props

```diff
@@ -0,0 +1,2 @@
+<Project>
```

Comment:
Why is it necessary to completely isolate these directories (not just test/testassets) from the repo's configuration? concern here is maintenance headache of spreading the gRPC versions around. Even for test/testassets, we usually _generate_ a file containing the versions from eng/Versions.props i.e. those projects are _mostly_ isolated from the rest of the respo.

nit: If these Directory.Build.* files are truly required, use `<Project />`.

Reply:
Update test project to not be completely isolated.

Client/server use shared runtime.